src/main/java/com/groupbyinc/api/Query.java

```diff
@@ -1193,4 +1186,72 @@ public Query addQueryUrlParams(String key, String value) {
         this.queryUrlParams.put(key, value);
         return this;
     }
+
+    /**
```

Comment:
@willwarren can you please review this section. it is the initial documentation for query-time biases